src/neo4j_graphrag/experimental/components/types.py

```diff
@@ -170,3 +171,10 @@ def lexical_graph_node_labels(self) -> tuple[str, ...]:
 class GraphResult(DataModel):
     graph: Neo4jGraph
     config: LexicalGraphConfig
+
+
+class SchemaEnforcementMode(str, Enum):
+    NONE = "none"
+    STRICT = "strict"
+    # future possibility: OPEN = "open" -> ensure conformance of nodes/props/rels that
```

Comment:
I'd remove this comment, I understand the point but no one knows when we will implement it so it can stay like this for a long time :-)